### ISSUE TITLE
Does this accomplish SSO between multiple TLDs?

### ISSUE THREAD

**Issue by softrare** [korotovsky/SingleSignOnServiceProviderBundle#18]
Hi! I am actually a little confused what this accomplishes. We have multiple systems - all running on Symfony 4, all running on different TLDs - one of which being an identity provider system being logged into through OAuth2 (FOSOAuthServerBundle). What can this software accomplish for us? Can it do SSO in the sense of that if you're logged into one website, you're automatically also logged in another? Sorry for this question, but I was really missing information about this. Thank you in advance!

**Comment by korotovsky** [comment]
It can handle different tld but not automatically, on sp you must initiate authentication process somehow

**Comment by softrare** [comment]
Thank you! You mean it must somehow recognize the session id or or have another clue about whom to log in right?

**Comment by korotovsky** [comment]
IdP passes token to SP and then SP makes trusted Server-Side request when user is on the SP to get additional data and validate this token. Then SP authenticates user.

**Comment by softrare** [comment]
Yes, but how does SP know who is that user if he only ever gets the user information from IdP? Or does the SP need his own login form in this scenario? Because right now we're referring the user exclusively to IdP for that.

**Comment by korotovsky** [comment]
IdP returns all the data by token, username, roles and so on in the trusted server-2-server request
